engine/src/test/scala/cromwell/MetadataWatchActor.scala

```diff
@@ -18,6 +18,7 @@ final case class MetadataWatchActor(promise: Promise[Unit], matchers: Matcher*)
 
   override def receive = {
     case PutMetadataAction(events) if unsatisfiedMatchers.nonEmpty =>
+      println(events.map(_.value.map(_.value)))
```

Comment:
I'm curious if this was intentionally added?

Reply:
I think this was @Horneth ?

Reply:
it was intentionally added but unintentionally not removed 😄 
It was to help me debug some failing tests, it can be reverted